core/services/fluxmonitorv2/orm_test.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/stretchr/testify/require"
 
+	commontxmgrmocks "github.com/smartcontractkit/chainlink/common/txmgr/types/mocks"
```

Comment:
This package alias is pretty gnarly. Not sure I have any better suggestions though :thinking:

Reply:
at the very least it could be `commontxmmocks` ...could shorten it further to `comtxmmocks`?